examples/with-route-as-modal/components/PostCardGrid.js

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link'
+
+export const posts = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+export default function PostCardGrid() {
+  return (
+    <div className="postCardGridWrapper">
+      <h2>With QueryString Routing</h2>
```

Comment:
@lfades Let me know if this is close to what you had in mind here https://github.com/zeit/next.js/pull/12945#issuecomment-633684632

Reply:
Yes